scheduler/replica_scheduler_test.go

```diff
@@ -239,6 +239,9 @@ func (s *TestSuite) TestReplicaScheduler(c *C) {
 		TestDiskID1: {
 			StorageAvailable: TestDiskAvailableSize,
 			StorageScheduled: 0,
+			Conditions: map[types.NodeConditionType]types.Condition{
+				types.NodeConditionTypeDiskSchedulable: newNodeCondition(types.NodeConditionTypeDiskSchedulable, types.ConditionStatusTrue),
```

Comment:
All those condition settings here, and in the previous tests, should be easily swapped for a short function. But that can wait until we got this done first.